### DISCUSSION TITLE
spotlessApply the rest?

### DISCUSSION THREAD

**Discussion by tonygermano** [mozilla/rhino#1024] [Q&A] [answered]
Is there any reason to not reformat the remaining main (not test) source files at this point just to get it out of the way? A good number of the files have already been done.

I think most of the old PRs and branches with large change sets that were created before spotless was required have been resolved now or likely contain files that have already been reformatted. I think it would make it easier to submit and review PRs going forward if they would otherwise require a separate spotless commit.

**Comment by p-bakker** [comment]
I'd say YES, go for it

**Comment by rbri** [comment]
+ 1

**Comment by gbrail** [comment, marked as answer]
I could see doing that. I was hoping to avoid giant commits and unnecessary
changes for future generations of maintainers, but we've already
reformatted so many hairy 20-year-old source files that perhaps we should
do the rest.

But no, not the tests yet as I fear that would generate a truly gigantic
number of changes.

**Reply by rbri** [reply]
did a quick check - we have still 90 files in the src folder that will change
and 28 in testsrc

**Comment by p-bakker** [comment]
Think we can conclude the answer is a resounding Yes.

@tonygermano Will you pick up this task?

**Comment by carlosame** [comment]
This could be done as part of #1092.

**Reply by p-bakker** [reply]
Wouldn't it be better to do it as a separate PR, as to not mix things?

**Reply by carlosame** [reply]
> Wouldn't it be better to do it as a separate PR, as to not mix things?

Yes, but IMHO as long as the reformatting comes as a separate commit, the specific PR does not matter a lot. And the PR for #1092 would affect the entire codebase as well.

**Reply by p-bakker** [reply]
Unless the PR gets squashed during merge.

Also: getting the reformatted code into master as a separate PR is probably going to happen a lot faster than the Pr for #1092 (although I have no idea exactly what that one will entail)

**Comment by tuchida** [comment]
ref. #1159 Apply spotlessApply to all code.

**Comment by p-bakker** [comment]
Locking this discussion as this got done via #1159
